lib/galaxy/datatypes/text.py

```diff
@@ -911,3 +912,115 @@ def _looks_like_yaml(self, file_prefix: FilePrefix):
             except yaml.YAMLError:
                 return False
             return False
+
+
+@build_sniff_from_prefix
+class Castep(Text):
```

Comment:
All three of these might be thematically better placed in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/molecules.py as they relate to a particular molecular material modelling software, but then these files are just text logs/YAML-style parameters, rather than molecular structure files like Cell, CIF, PDB and co. 
I'll let the core devs weigh in on this...